### PR TITLE
OCM-4458 | feat: add new parameter to enable external auth config

### PR DESCRIPTION
**Pull request by chenz4027**
https://issues.redhat.com/browse/OCM-4458

### CODE REVIEW

File: cmd/create/cluster/cmd.go [hunk 1]
```diff
@@ -351,6 +353,14 @@ func initFlags(cmd *cobra.Command) {
 	)
 	flags.MarkHidden(ClassicOidcConfigFlag)
 
+	flags.BoolVar(
+		&args.externalAuthProvidersEnabled,
+		ExternalAuthProvidersEnabledFlag,
+		false,
+		"Enable external auth configuration of a Hosted Control Planes.",
```
Comment (philipwu08) [inline cmd/create/cluster/cmd.go:360]: Maybe not mix singular and plural?

"Enable external auth configuration for a Hosted Control Plane cluster."

**Comment by lucasponce** [comment]
Minor comments to unify the wording in the new flag.

File: cmd/create/cluster/cmd.go (hunk 1 above)
Comment (chenz4027) [reply cmd/create/cluster/cmd.go:360]: Thanks, will address that

**Comment by chenz4027** [comment]
> Minor comments to unify the wording in the new flag.
Changed the texts

**Comment by lucasponce** [comment]
LGTM

**Review by ciaranRoche — APPROVED** [review]
/lgtm

**Pull request merged**